mobile/src/main/java/ch/epfl/sdp/mobile/ui/puzzles/PuzzleInfoState.kt

```diff
@@ -0,0 +1,24 @@
+package ch.epfl.sdp.mobile.ui.puzzles
+
+import androidx.compose.runtime.Stable
+import ch.epfl.sdp.mobile.application.chess.Puzzle
+
+/** An interface that represents the user interface information about the puzzle. */
+@Stable
+interface PuzzleInfoState {
+  /** The [Puzzle] that is loaded */
+  val puzzleInfo: PuzzleInfo
+
+  /** The state of the puzzle */
+  var puzzleState: PuzzleState
+
+  var currentMoveNumber: Int // Always start with playing "computer" move
+
+  var expectedMoves: Int
```

Comment:
No Documentation

Reply:
Fixed in https://github.com/epfl-SDP/android/pull/309/commits/3fdd0c6838e04702271df86b02692073e9a8dd9e